files/image_config/constants/constants.yml

```diff
@@ -29,7 +29,6 @@ constants:
         v6:
           - "deny 0::/0 le 59"
           - "deny 0::/0 ge 65"
-          - "permit fe80::/64"
```

Comment:
why remove, cannot find the description for this removal.

Reply:
Answered by email